AI/VCAI/AIUtility.cpp

```diff
@@ -431,18 +430,31 @@ bool isBlockVisitObj(const int3 & pos)
 	return false;
 }
 
-int howManyTilesWillBeDiscovered(const int3 & pos, int radious, CCallback * cbp)
+bool hasReachableNeighbor(const int3 &pos, const CPathsInfo* paths, CCallback * cbp)

```

Comment:
I do not like mutable CCallback  pointer here.

Reply:
Sorry, now I call non-const method CCallback::getPathsInfo(hero)